tracing/jersey/src/main/java/io/helidon/tracing/jersey/AbstractTracingFilter.java

```diff
@@ -166,8 +167,16 @@ public void filter(ContainerRequestContext requestContext, ContainerResponseCont
         requestContext.setProperty(SPAN_FINISHED_PROPERTY, true);
         span.finish();
 
-        Context context = Contexts.context().orElseThrow(() -> new IllegalStateException("Context must be available in Jersey"));
-        context.get(Scope.class).ifPresent(Scope::close);
+        // using the helidon context is not supported here, as we may be executing in a completion stage future returned
+        // from a third party component
+        // Context context = Contexts.context().orElseThrow(() ->
```

Comment:
Do we want to leave the old commented out like this?

Reply:
I wanted to have something there, so we do not refactor it to Helidon context. Not sure if all is needed.

Reply:
Leave the comment without the old code maybe? "// using the Helidon context Context.context() is not supported here ..."

Reply:
ok